test/end-to-end/signup.test.js

```diff
@@ -134,10 +134,18 @@ describe("SuperTokens SignUp", function () {
             await page.evaluate(() => window.SuperTokens.redirectToAuth());
```

Comment:
it would be good to add / modify tests that actually do redirection backend and see that that preserves query param in the case of:
- email password login
- third party login
- passwordless magic linlk
- passwordless otp
- email verification

There should already be tests for these somewhere where we actually check that it redirects back properly, so you can just modify those to add query params and check that they are preserved.

Reply:
- redirectBack: false test
- redirect where path is `/`, but has only query params.

Reply:
done